config/routes.rb

```diff
@@ -1,3 +1,19 @@
 Rails.application.routes.draw do
+  root 'events#index'
+  get 'signup' => 'users#new', as: 'signup'
+  get 'login' => 'sessions#new', as: 'login'
+  get 'logout' => 'sessions#destroy', as: 'logout'
```

Comment:
- In RESTFUL APIs we don't need to overly complicate/customize route names:
     - So user signup is simply included here:  `resources :users`
     - login in is simply: `resources :sessions, only: [:create]`
     - And logout is simply: `resources :sessions, only: [:create]`